discovery-leap-of-faith.md

```diff
@@ -0,0 +1,38 @@
+
+# Discovery using the cloud and leap-of-faith
```

Comment:
Right! Good points. "leap of faith" refers to "trust on first use" - when the client first connects to what it hopes is the box, it has no evidence that this host actually is the box it's looking for, except for the fact that it's the only visible candidate. See e.g. https://www.ietf.org/mail-archive/web/btns/current/msg00789.html. It's also how ssh establishes trust.

The purpose of this discovery is for the client (e.g. the smartphone of the user) to connect to the foxbox. This includes:
* discovering an IP address on which to reach the foxbox
* discovering the TLS server certificate of the foxbox
* possibly exchanging (password) credentials for a session (access) token